pkg/controller/flightctl/flightctl.go

```diff
@@ -182,7 +225,7 @@ func (f *FlightCtlManager) getFlightCtlClientServiceAccountToken(ctx context.Con
 	}
 
 	// Get the token using TokenRequest API
-	tokenResponse, err := f.clientHolder.KubeClient.CoreV1().ServiceAccounts(FlightCtlNamespace).
+	tokenResponse, err := f.clientHolder.KubeClient.CoreV1().ServiceAccounts(os.Getenv("POD_NAMESPACE")).
```

Comment:
I am not sure why we do this. If it is the same namespace, can we directly use the import controller's sa and give it sufficient permission to access flightctl. So that we do not need to call api to get token, but read from file

Reply:
The token will be delivered to flightctl-agent to access agentresgistraion endpoint of the import-controller, so we create token with specific 7 days duration to avoid credentials.

We used to bind 2 clusterroles with the serviceaccount `flightctl-client`, but now it seems a bug, I will seperate the clusterrolebinding.

Reply:
The clusterrolebinding sperated, now it's:
* `flightctl-agent-registration` <-> `flightctl-agent-registration`:  used in the flightctl Repository, will be delivered to the flightctl-agent in managed cluster side.
* `managedcluster-import-controller-v2` <-> `flightctl-client`: used for import-controller to access the flightctl-api service on the hub side.